plugins/GSdx/GSWndWGL.cpp

```diff
@@ -323,16 +331,45 @@ void* GSWndWGL::GetProcAddress(const char* name, bool opt)
 
 //TODO: check extensions supported or not
 //FIXME : extension allocation
-void GSWndWGL::SetVSync(int vsync)
+void GSWndWGL::SetSwapInterval(int vsync)
 {
 	// m_swapinterval uses an integer as parameter
 	// 0 -> disable vsync
 	// n -> wait n frame
 	if (m_swapinterval) m_swapinterval(vsync);
 }
 
+bool GSWndWGL::CompositorEnabled()
+{
+	BOOL dwm;
+	if (DwmIsCompositionEnabled(&dwm) != S_OK)
+		return false;
+
+	// Note: DWM is always true on Win8 and later
+	if (!dwm)
+		return false;
+
+	// Easy case are done. It remains driver black magic
+
+	// Recent driver (+380.xx ??) seem to always get v-sync enforced someway.
+	// XXX it would be nice to understand what happen to fix it properly
+	if (GLLoader::nvidia_buggy_driver)
+		return true;
```

Comment:
Hmm. I think this means that the standard and adaptive options for Windows 8+ NVIDIA users is always the same?